src/answers-umd.js

```diff
@@ -172,6 +173,10 @@ class Answers {
     masterSwitchApi.isDisabled(parsedConfig.apiKey, parsedConfig.experienceKey)
       .then(isDisabled => !isDisabled && this._initInternal(parsedConfig, globalStorage, persistentStorage))
       .catch(() => this._initInternal(parsedConfig, globalStorage, persistentStorage));
+
+    if (!parsedConfig.disableCssVariablesPolyfill) {
+      this.polyfillCssVariables();
```

Comment:
Do we need to define onError and onSuccess handlers here that invoke onReady? If we don't could there be a race condition?